[2025-04-20T22:53:40.878+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-20T22:53:40.899+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_pipeline.meteo_operations.collect_meteo_data manual__2025-04-20T22:53:35.945068+00:00 [queued]>
[2025-04-20T22:53:40.908+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_pipeline.meteo_operations.collect_meteo_data manual__2025-04-20T22:53:35.945068+00:00 [queued]>
[2025-04-20T22:53:40.909+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-20T22:53:40.919+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): meteo_operations.collect_meteo_data> on 2025-04-20 22:53:35.945068+00:00
[2025-04-20T22:53:40.927+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'weather_data_pipeline', 'meteo_operations.collect_meteo_data', 'manual__2025-04-20T22:53:35.945068+00:00', '--job-id', '712', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpg4yhoocj']
[2025-04-20T22:53:40.929+0000] {standard_task_runner.py:105} INFO - Job 712: Subtask meteo_operations.collect_meteo_data
[2025-04-20T22:53:40.933+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2440) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-20T22:53:40.934+0000] {standard_task_runner.py:72} INFO - Started process 2458 to run task
[2025-04-20T22:53:40.980+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_data_pipeline.meteo_operations.collect_meteo_data manual__2025-04-20T22:53:35.945068+00:00 [running]> on host 3ec408313633
[2025-04-20T22:53:41.066+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_data_pipeline' AIRFLOW_CTX_TASK_ID='meteo_operations.collect_meteo_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-20T22:53:35.945068+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-20T22:53:35.945068+00:00'
[2025-04-20T22:53:41.067+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-20T22:53:41.108+0000] {python.py:240} INFO - Done. Returned value was: {'latitude': 21.4901, 'longitude': 39.1862, 'timestamp': '2025-04-20 00:00:00', 'temperature': 27.223501205444336, 'humidity': 69.0, 'wind_speed': 4.0249223709106445, 'pm2_5': 91.30000305175781, 'co': 199.0}
[2025-04-20T22:53:41.133+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-20T22:53:41.134+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=meteo_operations.collect_meteo_data, run_id=manual__2025-04-20T22:53:35.945068+00:00, execution_date=20250420T225335, start_date=20250420T225340, end_date=20250420T225341
[2025-04-20T22:53:41.189+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-20T22:53:41.214+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-20T22:53:41.216+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
